Dockerfile-tools.template

```diff
@@ -0,0 +1,3 @@
+FROM rust:%%RUST-IMAGE%%
+
+RUN rustup component add rustfmt clippy llvm-tools-preview 
```

Comment:
@Christiaan676 I think this is now available as `llvm-tools` directly without the preview

Reply:
This indeed seems to be the case. (https://rust-lang.github.io/rustup/concepts/components.html). 

This merge request is already more than a year old. I'm willing to update it and change things if needed, but only if a maintainer is wiling to review and eventually merge it. Currently updating it seems to be a wast of my time.